releases/release-6.5.8.md

```diff
@@ -0,0 +1,150 @@
+---
+title: TiDB 6.5.8 Release Notes
+summary: 了解 TiDB 6.5.8 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 6.5.8 Release Notes
+
+发版日期：2024 年 x 月 x 日
+
+TiDB 版本：6.5.8
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.5.8#version-list)
+
+## 兼容性变更
+
+- note [#issue](https://github.com/pingcap/${repo-name}/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+- (dup): release-7.1.2.md > 兼容性变更 - 在安全增强模式 (SEM) 下禁止设置 [`require_secure_transport`](https://docs.pingcap.com/zh/tidb/v7.1/system-variables#require_secure_transport-从-v610-版本开始引入) 为 `ON`，避免用户无法连接的问题 [#47665](https://github.com/pingcap/tidb/issues/47665) @[tiancaiamao](https://github.com/tiancaiamao)
```

Comment:
```suggestion
```
@tiancaiamao This note was included in v6.5.6. Can we remove this from v6.5.8 release notes?
https://github.com/pingcap/docs-cn/blob/217405f7bfd0e121c96e0f4d103ade7401e29a43/releases/release-6.5.6.md?plain=1#L16